run.py

```diff
@@ -412,7 +412,7 @@ def fit(self):
 				if kill_cnt % 10 == 0 and self.p.gamma > 5:
 					self.p.gamma -= 5 
 					self.logger.info('Gamma decay on saturation, updated value of gamma: {}'.format(self.p.gamma))
-				if kill_cnt > 25:
+				if kill_cnt > 501:
```

Comment:
I would prefer implementing an parameter here as well, so we can easily change the number of epochs if e.g. requested by tong

Reply:
I added two parameters:

1. A parameter to enable early stopping if needed.
2. A parameter to set the amount of epochs for early stopping, default is 25